e-commerce_fake-store/src/pages/cart-page/index.js

```diff
@@ -0,0 +1,46 @@
+import Navbar from "../../components/navbar";
+import Product from "../../components/product-card";
+import { Link } from "react-router-dom";
+
+const Cart = () => {
+  let count = localStorage.getItem("cartitem");
+  const allData = [];
+  for (let i = 0; i < count; i++) {
+    const data = {
+      name: localStorage.getItem("itemname" + i),
```

Comment:
Again, as reviewed above. Remove the use of multiple local storages and combine them into a single object.

@shubhamshukla91